sdk/v1/client.go

```diff
@@ -68,32 +68,35 @@ func (cli *Client) Checker(ctx context.Context, check *health.CheckState) error
 }
 
 // PostJob creates a new reindex job for processing
-func (cli *Client) PostJob(ctx context.Context, headers client.Headers) (*models.Job, error) {
+func (cli *Client) PostJob(ctx context.Context, reqHeaders client.Headers) (*client.RespHeaders, *models.Job, error) {
 	var job models.Job
-	if headers.ServiceAuthToken == "" {
-		headers.ServiceAuthToken = cli.serviceToken
+	if reqHeaders.ServiceAuthToken == "" {
+		reqHeaders.ServiceAuthToken = cli.serviceToken
 	}
 
 	path := cli.hcCli.URL + "/" + cli.apiVersion + jobsEndpoint
-	_, b, err := cli.callReindexAPI(ctx, path, http.MethodPost, headers, nil)
+	respHeader, b, err := cli.callReindexAPI(ctx, path, http.MethodPost, reqHeaders, nil)
 	if err != nil {
-		return &job, err
+		return nil, &job, err
 	}
 
 	if err = json.Unmarshal(b, &job); err != nil {
-		return &job, apiError.StatusError{
+		return nil, &job, apiError.StatusError{
```

Comment:
```suggestion
		return nil, nil, apiError.StatusError{
```

Reply:
Yes I wasn't sure what to do here when I was updating the PostJob function. I put nil here at first but then the unit tests failed where they checked "Then the expected empty job is returned". And so I decided to leave the empty job in as the unit tests were expecting it. Perhaps we should check with Nathan as he wrote the PostJob function originally?

Reply:
done